crossdock/docker-compose.yml

```diff
@@ -22,7 +22,8 @@ services:
 
             - REPORT=compact
     go:
-        image: jaegertracing/xdock-go
+        # TODO Use the master branch once this branch is merged
+        image: jaegertracing/xdock-go:upgrade-crossdock-tests-for-native-opentracing-in-tchannel
```

Comment:
the Go repo has been upgraded, you can use the latest image now (rm the tag ^^^)